mmv1/templates/terraform/pre_create/datastream_connection_profile.go.tmpl

```diff
@@ -0,0 +1,18 @@
+{{/*
+	The license inside this block applies to this file
+	Copyright 2024 Google Inc.
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+*/ -}}
+
+_, exists := d.GetOk("create_without_validation")
+if !exists {
+	url += "false"
+	d.Set("create_without_validation", false)
+}
```

Comment:
Sorry for the back and forth here, but after reviewing [a discussion about `GetOkExists`](https://discuss.hashicorp.com/t/terraform-sdk-usage-which-out-of-get-getok-getokexists-with-boolean/41815) I think our best approach is actually to get the url parameters out of the base URL entirely.

So that would mean deleting `&force={{create_without_validation}}` from the `create_url` and calling `transport.AddQueryParams` here with something like `map[string]string{"force": "true"}` only if `d.Get("create_without_validation")` returns `true`.

We might also want to make sure there's at least one test that creates the resource with the field set to false or unspecified.

Reply:
Is the `force` url parameter required in the API endpoint?

Reply:
It would also be a breaking change if `force=false` has different behavior than omitting `force` altogether. Keeping it as is should be safest.